tests/test_do_ard.py

```diff
@@ -24,24 +68,32 @@ def test_do_ard(tmp_path):
         "walltime": "02:30:00",
         "workdir": "scratch_ls/",
         "workers": None,
-        "yamls_dir": ""
+        "yamls_dir": "",
     }
     usgs_level1_files = None
-    l1_zips = ['scene1.zip', 'scene2.zip',  'scene3.zip']
+    l1_zips = ["scene1.zip", "scene2.zip", "scene3.zip"]
     l1_count = len(l1_zips)
-    uuids2archive = ['uuid1', 'uuid2', 'uuid3']
+    uuids2archive = ["uuid1", "uuid2", "uuid3"]
     jobdir = tmp_path / "jobdir"
     jobdir.mkdir()
     run_ard = False
-    do_ard(ard_click_params,
-           l1_count,
-           usgs_level1_files,
-           uuids2archive,
-           jobdir,
-           run_ard,
-           l1_zips)
+    do_ard(
+        ard_click_params,
+        l1_count,
+        usgs_level1_files,
+        uuids2archive,
+        jobdir,
+        run_ard,
+        l1_zips,
+    )
 
     # Not checking the values, just that the files exist
-    assert os.path.exists(tmp_path / "jobdir" / ARCHIVE_FILE), ARCHIVE_FILE + ' does not exist.'
-    assert os.path.exists(tmp_path / "jobdir" / ODC_FILTERED_FILE), ODC_FILTERED_FILE + ' does not exist.'
-    assert os.path.exists(tmp_path / "jobdir" / PBS_ARD_FILE), PBS_ARD_FILE + ' does not exist.'
+    assert os.path.exists(tmp_path / "jobdir" / ARCHIVE_FILE), (
+        ARCHIVE_FILE + " does not exist."
+    )
+    assert os.path.exists(tmp_path / "jobdir" / ODC_FILTERED_FILE), (
```

Comment:
Recommendation: I think it's safer to join the path elements using os.path.join()

Reply:
This is ok for Path objects.